### PR TITLE
Volume

### LINKED ISSUE #1045: Write a Julia programme to find volume of cone
### Description

Write a programme to find volume of cone

Volume of cone = 1/3 * π * (radius)^2 * height

```
radius : 2
height : 3

volume : 12.56637
```

### PR DESCRIPTION
**Pull request by SpirosArk**
### Why:

Closes #1045 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/codinasion/codinasion/blob/master/contributing/self-review.md).

### CODE REVIEW

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Program that accepts a char as an input and returns it's position in alphabet
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:1]:
```suggestion
//Programme that accepts a char as an input and returns it's position in alphabet
```

File: programme/volume-of-cone/volume-of-cone.jl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Program to calculate volumeOfCone
```
Comment (KikiManjaro) [inline programme/volume-of-cone/volume-of-cone.jl:1]:
```suggestion
#Programme to calculate volumeOfCone
```

**Review by KikiManjaro — CHANGES_REQUESTED** [review]
Minor fixes related to #1295

**Review by KikiManjaro — APPROVED** [review]

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    if (input >= 97){
+        
+        position = input - 'a' + 1;
+        
+    }
+    else{
+    
+        position = input - 'A' + 1;
+    
+    }
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:20]:
```suggestion
    if (input >= 97){
        position = input - 'a' + 1;
    }
    else{
        position = input - 'A' + 1;
    }
```

**Comment by KikiManjaro** [comment]
You've made a mistake and modified index-of-alphabet in this MR, this one should only refer to volume

**Comment by PrajwalBorkar** [comment]
> You've made a mistake and modified index-of-alphabet in this MR, this one should only refer to volume

Actually it is showing here because other pr is not merged which contains this file.

**Comment by SpirosArk** [comment]
What should I change? I am confused right now...

**Comment by KikiManjaro** [comment]
@SpirosArk Don't worry, everything will be ok when your other MR will be merged 👌

For the next time just think about creating a new branch from master for each programs you add, the new code you add from a MR that is not merged yet should not be in an other MR that solves an other issue

**Comment by PrajwalBorkar** [comment]
> What should I change? I am confused right now...

No need to do anything now, just make sure that you include only one code in one branch e.g. in this volume branch includes changes related to volume code instead of having other files which is the alphabet.c file.

**Comment by SpirosArk** [comment]
Ok, I understand. Each file commit in different branch. Thank you!

**Review by PrajwalBorkar — APPROVED** [review]
I think this is ready to be merged. Thanks for the changes :)

**Review by harshraj8843 — APPROVED** [review]
👍🏻

**Comment by harshraj8843** [comment]
> Ok, I understand. Each file commit in different branch. Thank you!

Not necessarily each file in a different branch, but we should create solutions for different programmes in different branches

So, we can put the `solution` file and updated `README.md` file in one branch and then create a PR

In this way, PR(s) are easy to review

**Comment by SpirosArk** [comment]
When I update the Readme it asks me to create new branch. It does not accept to commit the change to an already existing branch. Let's say we have created earlier volume.c, when I will change the volume Readme it will prompt me to commit either on master or on another branch. This is where I mess up

**Comment by harshraj8843** [comment]
> When I update the Readme it asks me to create new branch. It does not accept to commit the change to an already existing branch. Let's say we have created earlier volume.c, when I will change the volume Readme it will prompt me to commit either on master or on another branch. This is where I mess up

okay, you are adding files directly to github (I thought you are committing from local)

In this case, you can create different files to different branches 👍🏻 

> **Note** Please create a new branch from `master`, so that PR(s) will not contain files from different PR(s)

**Comment by SpirosArk** [comment]
These days I am on vacation so I commit through cell phone written scripts. When I am at my local machine I do it as I should. With puling -commiting- pushing

**Pull request merged**